server/routes/temporary-accommodation/index.ts

```diff
@@ -0,0 +1,18 @@
+/* istanbul ignore file */
```

Comment:
Today I learned. An instruction to exclude this file from test coverage?

Reply:
Seems to be - this is following the convention of the AP routes